se/data/templates/compatibility.css

```diff
@@ -31,8 +31,8 @@ img[epub|type~="se:image.color-depth.black-on-transparent"][epub|type~="z3998:pu
 	background: transparent !important;
 }
 
-/* Or if the device supports prefers-color-scheme. We’ll invert the image in core.css. */
-@media (prefers-color-scheme){
+/* Or if the device supports prefers-color-scheme. We’ll invert the image in core.css. RMSDK requires a target media as well as a state. */
```

Comment:
I'm not particularly interested to test what happens here in practice, but back in the day you should write media queries along these lines so that some browsers *cough*IE*cough* would fail gracefully this way while otherwise they'd basically drop everything that followed.

From glancing at the linked topic it's not immediately clear to me if this actually fully works as intended as the comment here implies (i.e., whether `prefers-color-scheme` is supported) or if it simply allows for failing gracefully.

Anyway, might be a nitpick, might be nothing. I just thought you should know. ^_^

Reply:
I'm not sure what you're getting at; do you mean that there's a problem with this commit?

Reply:
Not with this commit, but potentially with this comment. Apparently the media query in its old form did something pretty bad to this particular piece of software. Presumably everything following it was dropped. A few images that don't look quite right might be a little ugly, but you wouldn't say all books are broken because of it.

The comment implies that RMSDK supports `prefers-color-scheme` provided you use a media query in this form. Perhaps it does, but all I can reasonably gather from the linked post is that this way the rest of the stylesheet(s) doesn't fail.